docs/api/createAsyncThunk.mdx

```diff
@@ -89,6 +89,7 @@ An object with the following optional fields:
 
 - `condition`: a callback that can be used to skip execution of the payload creator and all action dispatches, if desired. See [Canceling Before Execution](#canceling-before-execution) for a complete description.
 - `dispatchConditionRejection`: if `condition()` returns `false`, the default behavior is that no actions will be dispatched at all. If you still want a "rejected" action to be dispatched when the thunk was canceled, set this flag to `true`.
+- `idGenerator`: a function to use when generating the `requestId` for the request sequence. Defaults to use [nanoid](https://www.npmjs.com/package/nanoid).
```

Comment:
```suggestion
- `idGenerator`: a function to use when generating the `requestId` for the request sequence. Defaults to use [nanoid](https://redux-toolkit.js.org/api/other-exports#nanoid).
```

We don't use all of nanoid, so we should just link back specifically to what we inline and re-export

Reply:
Yeah, but the link should go to the sibling Markdown file instead of the website directly :)

Reply:
i'm fired :joy:

Reply:
Now addressed